src/Metrics.Extensions/src/Internal/Microsoft.ApplicationInsights.Metrics/AccumulatorAggregator.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+using Microsoft.ApplicationInsights.Metrics.Extensibility;
+
+using Util = Microsoft.ApplicationInsights.Metrics.Extensions.Util;
+
+namespace Microsoft.ApplicationInsights.Metrics
+{
+    internal sealed class AccumulatorAggregator : MetricSeriesAggregatorBase<double>
+    {
+        private static readonly Func<MetricValuesBufferBase<double>> MetricValuesBufferFactory = () => new MetricValuesBuffer_Double(capacity: 500);
+
+        private readonly object _updateLock = new object();
+
+        private readonly bool _restrictToUInt32Values;
+
+        private double _min;
```

Comment:
we don't do underscore for private fields. Has stylecop complained? If no - we need a new naming rule here